variables.tf

```diff
@@ -78,20 +70,32 @@ variable "groups" {
   default     = null
 }
 
+variable "tags" {
+  type        = map(string)
+  description = "(Optional) A map of tags that will be applied to the created IAM policy."
+  default     = {}
+}
 
 # ------------------------------------------------------------------------------
 # MODULE CONFIGURATION PARAMETERS
 # These variables are used to configure the module.
 # See https://medium.com/mineiros/the-ultimate-guide-on-how-to-write-terraform-modules-part-1-81f86d31f024
 # ------------------------------------------------------------------------------
+
 variable "module_enabled" {
   type        = bool
   description = "(Optional) Whether to create resources within the module or not. Default is true."
   default     = true
 }
 
+variable "module_tags" {
+  type        = map(string)
+  description = "(Optional) A map of tags that will be applied to all created resources that accept tags. Tags defined with 'module_tags' can be overwritten by resource-specific tags."
+  default     = {}
+}
+
 variable "module_depends_on" {
   type        = any
-  description = "(Optional) A list of external resources the module depends_on. Default is []."
+  description = "(Optional) A list of external resources the module depends_on."
```

Comment:
doe snot match template afaik

Reply:
It does: https://github.com/mineiros-io/terraform-module-template/blob/main/variables.tf